external/esp_idf_port/esp32/fast_crypto_ops.c

```diff
@@ -0,0 +1,140 @@
+/******************************************************************
+ *
+ * Copyright 2019 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ******************************************************************/
+
+// Copyright 2015-2017 Espressif Systems (Shanghai) PTE LTD
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "crypto/common.h"
+#include "crypto/aes_wrap.h"
+#include "crypto/sha256.h"
+#include "crypto/crypto.h"
+#include "crypto/md5.h"
+#include "crypto/sha1.h"
+#include "crypto/aes.h"
+#include "crypto/dh_group5.h"
+#include "wps/wps.h"
+#include "wps/wps_i.h"
+#include "wpa2/eap_peer/eap.h"
+#include "wpa2/tls/tls.h"
+#include "wpa2/eap_peer/eap_methods.h"
+#include "wpa2/eap_peer/eap_i.h"
+#include "wpa2/eap_peer/eap_common.h"
+#include "esp_wifi_crypto_types.h"
+/*
+ * The parameters is used to set the cyrpto callback function for station connect when in security mode,
+ * every callback function can register as fast_xxx or normal one, i.e, fast_aes_wrap or aes_wrap, the
+ * difference between them is the normal API is calculate by software, the fast one use the hardware
+ * crypto in it, can be faster than the normal one, so the callback function register in default is which
+ * we recommend, so as the API in WPS default and WPA2 default.
+ */
+const wpa_crypto_funcs_t g_wifi_default_wpa_crypto_funcs = {
+	.size = sizeof(wpa_crypto_funcs_t),
+	.version = ESP_WIFI_CRYPTO_VERSION,
+	.aes_wrap = (esp_aes_wrap_t) aes_wrap,
```

Comment:
remove the space. please check all the lines in this file.

Reply:
done, keep tizenrt coding rule.